src/multiplots.ts

```diff
@@ -2035,34 +2084,38 @@ export class MultiplotCom {
     }
 
     public static frame_to_frame_communication(frame1, frame2) {
-      if (frame1.axes[0].name !== frame2.axes[0].name) return;
-      frame2.axes[0].rubberBand = frame1.axes[0].rubberBand;
-      frame2.axes[1].rubberBand = frame1.axes[1].rubberBand;
+      frame2.axes.forEach(axisFrame2 => {
+        frame1.axes.forEach(axisFrame1 => {
+          if (axisFrame1.name == axisFrame2.name && axisFrame1.name != 'number') {
```

Comment:
What does the "number" named frame stand for ?

Reply:
changed to currentFrame and ohterFrame

Reply:
Ok with the change (could even be `frame` and `other`), but what usecase denotes this check : `axisFrame1.name != 'number'` ?

Reply:
This denotes the case of Histogram number axis which must not be changed with it. I agree this is not the best way but it is the simplest and will change with multiplot refactor